airbyte-cdk/python/airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, source_config: ConnectionDefinition, debug: bool = False, con
         self.logger = logging.getLogger(f"airbyte.{self.name}")
 
         # Controls whether we build components using the manual handwritten schema and Pydantic models or the legacy flow
-        self.construct_using_pydantic_models = construct_using_pydantic_models
+        self.construct_using_pydantic_models = True
```

Comment:
Leaving it in actually so that we can still keep the legacy flow in the short term. But post-refactor we can clean this up and remove the old legacy flow and this conditional lock